tests/automotive-IMA/basicmath_large/Makefile

```diff
@@ -0,0 +1,18 @@
+CC		:=	riscv64-unknown-elf-gcc
+OBJDUMP := riscv64-unknown-elf-objdump --disassemble-all --disassemble-zeroes --section=.text --section=.text.startup --section=.data
+
+TARGET	:= basicmath_large.c rad2deg.c cubic.c isqrt.c
+GCC_OPTS = -m32 -Wa,-march=RV32IMA -std=gnu99 -msoft-float
```

Comment:
Oh ok! Makes sense! I think I didn't generate my compiler with "multilib" and this can make a big difference. I'll regenerate it.